sources/model/entity/User.ts

```diff
@@ -28,17 +29,22 @@ export class User {
 
     groups: Group[] = [];
 
+    @ManyToMany(type => Calendar, calendar => calendar.owners)
```

Comment:
Can a `Calendar` belong to several `User` (directly, without Group) ? I was expecting a `OneToMany` here

Reply:
Let's say we have a university calendar and want it to be shared between students, they will all be owners of the calendar but will not necessarily be in a group, no ?